packages/teams-js/test/public/sharing.spec.ts

```diff
@@ -1,10 +1,12 @@
 import { app } from '../../src/public/app';
 import { ErrorCode } from '../../src/public/interfaces';
+import { FrameContexts } from '../../src/public/constants';
 import { sharing } from '../../src/public/sharing';
 import { Utils } from '../utils';
 
 describe('sharing_v1', () => {
   const utils = new Utils();
+  const allowedContext = ['content', 'sidePanel', 'task', 'stage', 'meetingStage'];
```

Comment:
Can we name this variable `allowedContexts` since it's an array of contexts?

Reply:
Sure.

Reply:
Thanks! Please also update the same variable in the v2 tests.